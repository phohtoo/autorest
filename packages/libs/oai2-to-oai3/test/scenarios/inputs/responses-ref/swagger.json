{
  "swagger": "2.0",
  "info": {
    "version": "1.0",
    "title": "test",
    "description": "Test"
  },
  "produces": ["application/json"],
  "consumes": ["application/json"],
  "paths": {
    "/calling": {
      "delete": {
        "tags": ["Recording"],
        "summary": "Stop recording the call.",
        "operationId": "ServerCalls_StopRecording",
        "responses": {
          "401": {
            "$ref": "#/responses/401"
          }
        }
      }
    }
  },
  "responses": {
    "401": {
      "description": "Unauthorized",
      "schema": {
        "type": "object"
      }
    }
  }
}
