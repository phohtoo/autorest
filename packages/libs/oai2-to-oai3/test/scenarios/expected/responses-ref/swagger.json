{
  "servers": [],
  "openapi": "3.0.0",
  "info": {
    "version": "1.0",
    "title": "test",
    "description": "Test"
  },
  "paths": {
    "/calling": {
      "delete": {
        "tags": [
          "Recording"
        ],
        "summary": "Stop recording the call.",
        "operationId": "ServerCalls_StopRecording",
        "responses": {
          "401": {
            "$ref": "#/components/responses/401"
          }
        }
      }
    }
  },
  "components": {
    "responses": {
      "401": {
        "description": "Unauthorized",
        "content": {
          "application/json": {
            "schema": {
              "type": "object"
            }
          }
        }
      }
    }
  }
}